src/icons/clear.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
I'm totally fine with this for now, but we definitely need to figure out how we want to handle icons before we add more after this.